src/aap_eda/services/activation/manager.py

```diff
@@ -1052,14 +1058,23 @@ def _create_activation_instance(self):
         }
         args[f"{self.db_instance_type}"] = self.db_instance
         try:
-            models.RulebookProcess.objects.create(**args)
+            rulebook_process = models.RulebookProcess.objects.create(**args)
         except IntegrityError as exc:
             msg = (
                 f"Activation {self.db_instance.id} failed to create "
                 f"activation instance. Reason: {exc}"
             )
             self._error_activation(msg)
             raise exceptions.ActivationStartError(msg) from exc
+        queue_name = self._get_queue_name()
```

Comment:
Do we expect the queue name could change since line 1050?

Reply:
Nope. It is a mistake. Good catch. I removed the 1050.